WebRoot/messages/ZmMsg_ja.properties

```
@@ -4493,4 +4494,6 @@ toolbar = \u30c4\u30fc\u30eb\u30d0\u30fc
 checked = \u30c1\u30a7\u30c3\u30af\u3057\u305f
 unChecked = \u672a\u30c1\u30a7\u30c3\u30af
 nextYear = \u6765\u5e74
-previousYear = \u524d\u5e74
\ No newline at end of file
+previousYear = \u524d\u5e74
+syncingToMobileBasedOnRole = \u6ce8: \u30e2\u30d0\u30a4\u30eb\u3078\u306e\u540c\u671f\u306b\u306f\u7ba1\u7406\u8005\u307e\u305f\u306f\u30de\u30cd\u30fc\u30b8\u30e3\u30fc\u306e\u5f79\u200b\u200b\u5272\u304c\u5fc5\u8981\u3067\u3042\u308a\u3001\u30b5\u30d6\u30d5\u30a9\u30eb\u30c0\u30fc\u304c\u3042\u308b\u5834\u5408\u306f\u305d\u308c\u3082\u542b\u307e\u308c\u307e\u3059\u3002
```

Are the two `U+200B` zero width space intended?
If it is not clear, please attach screenshots with and without them.
If they are unnecessary, please remove them.
```suggestion
syncingToMobileBasedOnRole = \u6ce8: \u30e2\u30d0\u30a4\u30eb\u3078\u306e\u540c\u671f\u306b\u306f\u7ba1\u7406\u8005\u307e\u305f\u306f\u30de\u30cd\u30fc\u30b8\u30e3\u30fc\u306e\u5f79\u5272\u304c\u5fc5\u8981\u3067\u3042\u308a\u3001\u30b5\u30d6\u30d5\u30a9\u30eb\u30c0\u30fc\u304c\u3042\u308b\u5834\u5408\u306f\u305d\u308c\u3082\u542b\u307e\u308c\u307e\u3059\u3002
```